odpc.client/src/features/publicatie/PublicatieDetails.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <simple-spinner v-if="isFetching"></simple-spinner>
+
+  <form v-else aria-live="polite" @submit.prevent="submit">
+    <alert-inline v-if="error">Er is iets misgegaan, probeer het nogmaals...</alert-inline>
+
+    <fieldset v-else-if="publicatie">
```

Comment:
Als ik het goed begrijp zie je als er iets mis gaat alleen een foutmelding en een 'terug' knop. Dus als ik het nog een keer wil proberen moet ik eerst terug naar de lijst met publicaties en dan naar de publicatie gaan waar ik mee bezig was (of een nieuwe). Misschien is het handiger als je bij een foutmelding nog steeds het formulier en de opslaan knop kan zien?

Reply:
Patroontje gewijzigd zoals besproken (ook voor GebruiekrsgroepDetails): initiele get error krijgt inline alert met back button en na submit kan je nu retry doen.